playbooks/roles/common/vars/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 streisand_common_packages:
+  # Required for generating passwords (not pre-installed on Ubuntu minimal)
+  - dictionaries-common
+  - wamerican-huge
```

Comment:
The `wamerican-huge` package is already installed (it's the last entry in this alphabetical list). The `dictionaries-common` package is a dependency for `wamerican-huge`. I'm fine with adding `dictionaries-common` to this list, but APT should already be taking care of this. I'm not sure why that isn't the case?